packages/cli/src/const/options.ts

```diff
@@ -1,4 +1,4 @@
-import chalk from 'chalk';
+import * as chalk from 'chalk';
```

Comment:
Is `* as ` needed for anything?

Reply:
Yes, starting on 3.0 they have started to use default exports